assets/sass/config/_variables.scss

```diff
@@ -223,6 +225,10 @@ $c-entity-search-autocomplete-bg-hover: rgba($c-silver, 0.2);
 $c-setup-analytics-notice-bg: $c-alabaster;
 $c-setup-analytics-notice-text: $c-cape-cod;
 
+// Analtyics CTAs
+$c-analytics-cta-bg: $c-aqua-haze;
```

Comment:
@techanvil Maybe we should rename that variable to `$c-activate-analytics-cta-bg` since `$c-analytics-cta-bg` is a bit generic 🤔

Reply:
Hmm... my only thought there is it then sounds like it's specific to `ActivateAnalyticsCTA` whereas it's also used by `CreateGoalCTA` too... I don't really mind either way though, would you prefer I change it to `$c-activate-analytics-cta-bg`?

Reply:
I think `$c-search-funnel-analytics-cta-bg` would work :)

Reply:
Sounds good, I have made that change :+1: